### DISCUSSION TITLE
Soft wrap in WYSIWYG source code view

### DISCUSSION THREAD

**Discussion by AndreyKindin** [directus/directus#5976] [Feature Requests]
Hello,
is it possible to add an option with which WYSIWYG turns on a soft wrap in source code view.
Thank you.

**Comment by wc-matteo** [comment]
also in the `code` interface

**Reply by rijkvanzanten** [reply]
Seems like `codemirror` has a native `lineWrapping` boolean that can toggle this. Should be a pretty easy interface option to add if you wanna open a PR:

https://github.com/directus/directus/blob/b86dc14b55a68f38254e5f9b8e87069036588797/app/src/interfaces/input-code/index.ts#L37

https://github.com/directus/directus/blob/b86dc14b55a68f38254e5f9b8e87069036588797/app/src/interfaces/input-code/input-code.vue#L229

**Reply by wc-matteo** [reply]
yep, I was taking a look at that 👍🏿

**Reply by AndreyKindin** [reply]
is there anything new on this?

**Reply by wc-matteo** [reply]
for my part, I couldn't find the time... yet

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
